### PR TITLE
environment variable included

### PR DESCRIPTION
**Pull request by Ujjawal3**
This PR is related to issue #2.

### CODE REVIEW

**Comment by tusharkhatriofficial** [comment]
Could you please replace all the content of `.gitignore` with the contents of [this](https://github.com/github/gitignore/blob/main/Node.gitignore) file and please add you name at the very bottom of the README.md file

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,1056 @@
...
+      "dependencies": {
+        "body-parser": "^1.20.0",
+        "dotenv": "^16.0.3",
+        "ejs": "^2.7.4",
+        "express": "^4.18.1",
+        "lodash": "^4.17.21",
+        "mongoose": "^6.5.3"
+      }
+    },
+    "node_modules/@types/node": {
```
Comment (tusharkhatriofficial) [inline package-lock.json:20]: Could you please remove the unnecessary code from this file!

**Review by tusharkhatriofficial — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (Ujjawal3) [reply package-lock.json:20]: @tusharkhatriofficial I  guess it is not unnecessary code. It is automatically generated on installing dotenv package

**Comment by tusharkhatriofficial** [comment]
> Could you please replace all the content of `.gitignore` with the contents of [this](https://github.com/github/gitignore/blob/main/Node.gitignore) file and please add you name at the very bottom of the README.md file

.

File: package-lock.json (hunk 1 above)
Comment (tusharkhatriofficial) [reply package-lock.json:20]: Ya I just checked it.

**Comment by Ujjawal3** [comment]
@tusharkhatriofficial  I have done the required changes

**Pull request merged**